foyer/tests/implemented_opls_tests.txt

```diff
@@ -72,4 +72,7 @@ toluene
 trichloromethane
 2-methoxypropane
 methyltert-butylether
+t-butanol
+isopropanol
 methylnitroaniline
+
```

Comment:
```suggestion
```
Tests are failing since the driver (`test_opls` or something like that) is trying to type an atom with the name of a blank string